GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -592,6 +606,17 @@ public new void Load()
             LoadCustomDifftools();
         }
 
+        public new void Load(ref Dictionary<ObjectId, string> filePathByObjectId)
```

Comment:
`Dictionary` is a class, why is it passed by `ref`?

Reply:
see below